en/docs/guides/login/logging-into-simplesamlphp-using-is.md

```diff
@@ -0,0 +1,209 @@
+        `                         http://localhost/simplesaml                       `
+        to test the configured value.
+
+8.  Add a service provider to SimpleSAMLphp.
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi config/authsources.php 
+    ```
+
+    1.  Add the following section to the file and save.
+
+        ``` java
+        'wso2-sp' => array(  
+            
+        'saml:SP',  
+            
+        // The entity ID of this SP.  
+            
+        // Can be NULL/unset, in which case an entity ID is generated based on the metadata URL.  
+            
+        'entityID' => 'simplesaml',  
+            
+        // The entity ID of the IdP this should SP should contact.  
+            
+        // Can be NULL/unset, in which case the user will be shown a list of available IdPs.  
+            
+        'idp' => 'https://localhost:9443/samlsso',  
+            
+        // The URL to the discovery service.  
+            
+        // Can be NULL/unset, in which case a builtin discovery service will be used.  
+            
+        'discoURL' => NULL,  
+            
+        ),
+        ```
+
+        Here we assume WSO2 IS is running in localhost on 9443.
+
+9.  Add the identity provider metadata.
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi metadata/saml20-idp-remote.php 
+    ```
+
+    1.  Add the following section to the file and save.
+
+        ``` java
+        $metadata['https://localhost:9443/samlsso'] = array(  
+            
+            'name' => array(  
+            
+            'en' =>  'WSO2 IS',  
+            
+            'no' =>  'WSO2 IS',  
+            
+        ),  
+            
+            'description'   =>  'Login with WSO2 IS SAML2 IdP.',  
+            
+            'SingleSignOnService'  =>  'https://localhost:9443/samlsso',  
+            
+            'SingleLogoutService'  => 'https://localhost:9443/samlsso',  
+            
+            'certFingerprint'      => '6bf8e136eb36d4a56ea05c7ae4b9a45b63bf975d'  
+            
+        );
+        ```
+
+    2.  Note that metadata \['
+        `                           https://localhost:9443/samlsso                         `
+        '\] should match the value of 'idp' in step 8.
+
+    3.  Note that "6bf8e136eb36d4a56ea05c7ae4b9a45b63bf975d" is the
+        thumbprint of the default certificate ships with WSO2 IS. SAML2
+        Response is signed with this certificate.
+
+## Configure the service provider in WSO2 IS
+
+1.  [Start WSO2 Identity Server](insert-link). 
+
+2.  Sign in. Enter your username and password to log on to the
+    [Developer Portal](https://localhost:9443/developer-portal).
+
+3.  Select **Applications** from the left hand side panel. 
+
+4.  Click on **New Application**.
+
+5.  Click on **SAML Web Application**.
+
+6.  Fill in the **Name** and optionally, provide a brief
+    **Description** of the application.  
+
+7.  Click **Next**.
```

Comment:
fragment?

Reply:
Certain things like the images, assertion, issuer, etc can be different. So I don't think making it a fragment is a good idea.